Reactive Scheduler build error in version 2.7.0

../node_modules/@devexpress/dx-scheduler-core/dist/dx-scheduler-core.es.js
Module not found: Can't resolve 'rrule' in 'C:\work\cbd-phoenix\node_modules\@devexpress\dx-scheduler-core\dist'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\hiberfil.sys'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\pagefile.sys'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\swapfile.sys'
Compiling...
Failed to compile.

--- a/packages/dx-scheduler-core/package.json
+++ b/packages/dx-scheduler-core/package.json
@@ -63,6 +63,6 @@
   },
   "dependencies": {
     "moment": "^2.24.0",
-    "rrule": "^2.6.2"
+    "rrule": "2.6.4"
   }
 }

